test/node/conversation.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+const prep = require('../preparation');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let user;
+let client2;
+let user2;
+let conversation;
+let timeElapsed;
+describe('Conversation Tests', () => {
```

Comment:
In addition to these tests you also need to test other options (numberOfConversations, numberOfParticipants)